tests/schema/tests.py

```diff
@@ -1545,6 +1548,63 @@ def test_check_constraints(self):
         if not any(details['columns'] == ['height'] and details['check'] for details in constraints.values()):
             self.fail("No check constraint for height found")
 
+    @skipUnlessDBFeature('supports_column_check_constraints')
+    def test_remove_field_check_does_not_remove_custom_checks(self):
+        # Create table
+        with connection.schema_editor() as editor:
+            editor.create_model(Author)
+
+        # Add the custom check constraint
+        constraint = CheckConstraint(check=Q(height__gte=0), name='author_height_gte_0_check')
+        custom_constraint_name = constraint.name
+
+        try:
```

Comment:
Question: the `try...finally...` is just to ensure test isolation yes?

Reply:
yep, looks it unnecessary when state cleaned with last lines of test, so just removed it